kolibri/core/assets/src/heartbeat.js

```diff
@@ -70,12 +68,20 @@ export class HeartBeat {
   setInactive() {
     this.active = false;
   }
-  wait() {
+  get delay() {
```

Comment:
recommend prefixing internal functions and state with `_` so it's clear that these are not part of the class instance's public API

Reply:
Have addressed this in latest commits.